.github/actions/generate-invocation-ids/action.yml

```
@@ -0,0 +1,41 @@
+name: 'Generate invocation IDs for bazel'
+description: |
+    An action for generating uuids for bazel invocations in to provider easier way to find the
+    links to BuildBuddy.
+
+inputs:
+  targets:
+    description: |
+      YAML list of targets to generate UUIDs for. Each item must have:
+        name: display name (e.g. Build, Test)
+        output_id: GitHub Actions output variable
+      Example:
+        [{
+          "name": "Build",
+          "output_id": "build_id"
+        },{
+          "name": "Test",
+          "output_id": "test_id"
+        }]
+    required: true
+
+outputs:
+  output:
+    description: "JSON object containing all generated UUIDs"
+    value: ${{ steps.gen-uuids.outputs.output }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Generate Bazel invocation IDs
+      id: gen-uuids
+      shell: bash
+      run: |
+        set -euo pipefail
+
+        python ./.github/actions/generate-invocation-ids/generate-ids.py '${{ toJSON(fromJSON(inputs.targets)) }}'
```

I suggest moving the interpreter to a hash bang inside the script as suggested below.
```suggestion
       ./.github/actions/generate-invocation-ids/generate-ids.py '${{ toJSON(fromJSON(inputs.targets)) }}'
```